### DISCUSSION TITLE
Difference between Child rpc and Parent rpc while initializing POS client

### DISCUSSION THREAD

**Discussion by savannahar68** [0xPolygon/matic.js#296] [Q&A] [answered]
Hey guys new to Web3.0 world. I understand what providers are and how RPC work, however failing to understand the difference between Child and Parent Rpc needed to initialize POS client as shown here - https://github.com/maticnetwork/matic.js/blob/master/examples/utils.js

Thank you in advance for answering this basic question.

**Comment by ujjwalguptaofficial** [comment, marked as answer]
child RPC means - rpc for polygon and parent rpc means RPC for ethereum. You can initialize both or one rpc at a time. 

But when you are initiating a transaction on polygon chain, then child provider must be set and 

when you are doing a trasnaction on parent chain - then parent provider must be set.

**Reply by savannahar68** [reply]
Got it thanks a lot.
